sisyphus/simple_linux_utility_for_resource_management_engine.py

```diff
@@ -235,6 +240,10 @@ def submit_helper(self, call, logpath, rqmt, name, task_name, start_id, end_id,
         while True:
             try:
                 out, err, retval = self.system_call(sbatch_call)
+                if retval != 0:
+                    logging.warning(self._system_call_error_warn_msg(sbatch_call))
+                    time.sleep(gs.WAIT_PERIOD_BETWEEN_CHECKS)
+                    continue
```

Comment:
I don't think we should do this. There is already code below that handles submission errors

Reply:
Technically, the snippet right below handles timeouts, not errors. Both the `logging.warning` and the `time.sleep` are called with different arguments.

Reply:
I was referring to https://github.com/rwth-i6/sisyphus/blob/ce5f7a2cf70ceac7cf330fdb6e7ab3578892ef8c/sisyphus/simple_linux_utility_for_resource_management_engine.py#L248-L285

Reply:
Got it, thanks for the comment. I've now removed all references of return value checks in all actual queue submit commands.